packages/react-components/src/components/trace-context-component.tsx

```diff
@@ -265,7 +265,7 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
 
     private renderPlaceHolder() {
         return <div className='no-output-placeholder'>
-            {'Add outputs by clicking on an analysis in the trace explorer view'}
+            {'Trace file loaded successfully.\nTo see available analyses, view the Trace Explorer tab.'}
```

Comment:
The \n in the text does not work.

![placeholder_text](https://user-images.githubusercontent.com/4398897/104023179-43869a00-518f-11eb-8440-c6e982d20284.png)

I think you could just put 2 texts between { } and add a `<br>` in between

Reply:
Thanks for testing and catching that @tahini! Should be fixed now.